opencensus-exporter/src/main/java/com/spotify/ffwd/FfwdMetricsConverter.java

```diff
@@ -0,0 +1,182 @@
+/*-
+ * -\-\-
+ * opencensus-exporter
+ * --
+ * Copyright (C) 2016 - 2019 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.ffwd;
+
+import com.google.common.base.Strings;
+import io.opencensus.common.Function;
+import io.opencensus.common.Functions;
+import io.opencensus.exporter.metrics.util.MetricExporter;
+import io.opencensus.metrics.LabelKey;
+import io.opencensus.metrics.LabelValue;
+import io.opencensus.metrics.export.Distribution;
+import io.opencensus.metrics.export.Metric;
+import io.opencensus.metrics.export.MetricDescriptor.Type;
+import io.opencensus.metrics.export.Point;
+import io.opencensus.metrics.export.Summary;
+import io.opencensus.metrics.export.TimeSeries;
+import java.io.IOException;
+import java.net.SocketException;
+import java.net.UnknownHostException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class FfwdMetricsConverter extends MetricExporter {
+  private static final Logger LOG = LoggerFactory.getLogger(FfwdMetricsConverter.class);
+
+  private FastForward client;
```

Comment:
nit: the newlines are wonky here

Do we have linting on this project?

Reply:
linting is enable but that newline issue is a warning - i changed the linter to be more strict.